### PR TITLE
[MIG] project_timeline: Migration to 16.0

### PR DESCRIPTION
**Pull request by bvanjeelharia**

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+web
```
Comment (legalsylvain) [inline oca_dependencies.txt]: this file is not needed anymore in recent OCA branch.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Diff review. :+1: 

Minor blocking point.

Otherwise, LGTM.
thanks !

File: oca_dependencies.txt (hunk 1 above)
Comment (bvanjeelharia) [reply oca_dependencies.txt]: Removed the file as requested

**Review by legalsylvain — APPROVED** [review]
thanks !

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**